python_modules/libraries/dagster-azure/dagster_azure/blob/compute_log_manager.py

```diff
@@ -71,7 +72,7 @@ def __init__(
         self,
         storage_account,
         container,
-        secret_key=None,
+        secret_credentials,
```

Comment:
- [ ] resolve TypeError in dagster_azure_tests/blob_tests/test_compute_log_manager.py:252